### PR TITLE
libaom-av1: add v3.8.0, drop old versions

### PR DESCRIPTION
**Pull request by valgur**
.

### CODE REVIEW

**Review by ericLemanissier — APPROVED** [review]

File: recipes/libaom-av1/all/conanfile.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def build_requirements(self):
         if self.options.get_safe("assembly", False):
             self.tool_requires("nasm/2.15.05")
         if self._settings_build.os == "Windows":
-            self.tool_requires("strawberryperl/5.30.0.1")
+            self.tool_requires("strawberryperl/5.32.1.1")
```
Comment (perseoGI) [inline recipes/libaom-av1/all/conanfile.py:55]: Thanks, this will avoid potential conflicts

**Review by perseoGI — APPROVED** [review]
LGTM

File: recipes/libaom-av1/all/conanfile.py [hunk 2]
```diff
@@ -91,6 +91,10 @@ def package(self):
         cmake = CMake(self)
         cmake.install()
         rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        if self.settings.os == "Windows" and os.path.exists(os.path.join(self.package_folder, "lib", "aom_dll.lib")):
+            # Make the names of aom_dll.lib and aom.dll consistent with each other and other platforms
+            os.rename(os.path.join(self.package_folder, "lib", "aom_dll.lib"),
```
Comment (AbrilRBS) [inline recipes/libaom-av1/all/conanfile.py:96]: These renamos always come back to bite us in the most unexpected ways, so I'm always wary of them

Comment (uilianries) [reply recipes/libaom-av1/all/conanfile.py:96]: @valgur please, do not rename it. Follow what's expected from the upstream.

**Review by perseoGI — CHANGES_REQUESTED** [review]
Avoid renaming

File: recipes/libaom-av1/all/conanfile.py (hunk 2 above)
Comment (valgur) [reply recipes/libaom-av1/all/conanfile.py:96]: Uh, yes, that was not a great idea. I'll revert it.

**Review by AbrilRBS — APPROVED** [review]
Thanks!

**Review by perseoGI — APPROVED** [review]
LGTM now!

**Review by uilianries — APPROVED** [review]
LGTM

**Pull request merged**